doc-content/enterprise-only/admin-and-config/managing-business-central-artifacts-proc.adoc

```diff
@@ -0,0 +1,36 @@
+[id='managing-business-central-artifacts-proc']
+= Managing artifacts
+
+You can manage artifacts from the *Artifacts* page in {CENTRAL}. The artifact repository is a Maven repository and there is only one Maven repository for each installation.
+
+The *Artifacts* page lists all the artifacts in the Maven repository. You can also upload a new artifact to the Maven repository using the *Upload* button.
+
+[NOTE]
+=====
+Only _*.jar_, _*.kjar_, or _pom.xml_ files can be uploaded to the Artifacts repository.
+=====
+
+[float]
+==== Opening/viewing artifact
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Artifacts*.
+. Click *Open* to view the artifact details.
+. Click *Ok* to go back to the *Artifacts* page.
+
+[float]
+==== Downloading an artifact
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Artifacts*.
+. Click *Download*.
+. Browse to the directory where you want to save the artifact and click *Save*.
+
+[float]
+==== Uploading an artifact
+.Procedure
+. Log in to {CENTRAL}.
```

Comment:
missing instruction about how to proceed if your .jar is not a mavenized artifact

Reply:
Added note regarding it.